conda_build/__init__.py

```diff
@@ -21,3 +23,13 @@
     'sign',
     'skeleton',
 ]
+
+# Set default logging handler to avoid "No handler found" warnings.
+try:  # Python 2.7+
+    from logging import NullHandler
+except ImportError:
+    class NullHandler(logging.Handler):
+        def emit(self, record):
+            pass
+
+logging.getLogger(__name__).addHandler(NullHandler())
```

Comment:
Thanks for the comment.  I'm not sure why, but things seem to be working with the code as is.  Any insight?

I have a package on my channel, intentionally built with an 80-character prefix.  This recipe will try to bring it in as a dependency:

```
package:
  name: test-pkg
  version: 1.0

requirements:
  build:
    - conda-build-test-has-prefix-files
```

Now, if you build that recipe (``conda build -c msarahan .``), you see the logger output, as I would hope.  is there something else that isn't showing because of the way I have things now?

```
 msarahan@0109-msarahan-2  ~/code/conda-build/tests/test_pkg   logger  conda build -c msarahan .
BUILD START: test-pkg-1.0-0
updating index in: /Users/msarahan/miniconda2/conda-bld/osx-64
updating index in: /Users/msarahan/miniconda2/conda-bld/noarch

The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    conda-build-test-has-prefix-files-1.0|                0           3 KB  local

The following NEW packages will be INSTALLED:

    conda-build-test-has-prefix-files: 1.0-0 local

WARNING conda_build.build:create_env(648): Build prefix failed with prefix length 255
WARNING conda_build.build:create_env(649): Error was:
WARNING conda_build.build:create_env(650): Placeholder of length '80' too short in package local::conda-build-test-has-prefix-files-1.0-0.
The package must be rebuilt with conda-build > 2.0.
WARNING conda_build.build:create_env(651): One or more of your package dependencies needs to be rebuilt with a longer prefix length.
WARNING conda_build.build:create_env(653): Falling back to legacy prefix length of 80 characters.
WARNING conda_build.build:create_env(654): Your package will not install into prefixes > 80 characters.
```